sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -3602,6 +3914,52 @@ public partial class PrometheusForwarderDataSource : System.ClientModel.Primitiv
         string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.PrometheusForwarderDataSource>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
         System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.PrometheusForwarderDataSource>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
     }
+    public partial class Receiver : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Monitor.Models.Receiver>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.Receiver>
```

Comment:
Single word name is not allowed, could we add some prefix to specify it?

Reply:
Renamed to `PipelineGroupReceiver`